README.rst

```diff
@@ -0,0 +1,162 @@
+.. image:: ./util/images/craychapel.png
+
+
+.. _chapelhome-readme:
+
+The Chapel Language
+===================
+
+Chapel is an emerging programming language designed for productive parallel computing at scale. Chapel's design and implementation have been undertaken with portability in mind, permitting Chapel to run on multicore desktops and laptops, commodity clusters, and the cloud, in addition to the high-end supercomputers for which it was designed. Chapel's design and development are being led by `Cray Inc.`_ in collaboration with academia, computing centers, and industry. See `chapel.cray.com`_ for more information.
+
+.. _Cray Inc.: http://www.cray.com/
+.. _chapel.cray.com: http://chapel.cray.com/
+
+This is the 1.12.0 release of the Chapel compiler, intended to give
+potential users a look at what we're doing and the opportunity to
+provide us with feedback.  See the LICENSE file for the release's
+licensing terms.
+
+
+Quick Start
+-----------
+
+The following instructions are designed to get you up and running with
+Chapel with a minimum of fuss.  Note that building and using Chapel in
+this mode disables enhanced runtime options and optional language
+capabilities in the interest of a simple and clean build.
+
+
+0) See ``doc/prereqs.rst`` for more information about packages you
+   might need to have installed for Chapel to work.
+
+1) Make sure that your shell is in the directory containing this
+   ``README.rst`` file.  For example:
+
+    .. code-block:: sh
+
+        cd ~/chapel-1.12.0
+
+
+2) Set up your environment to use Chapel in "Quick Start" mode.
+
+
+======================================== ==========================================
+If you use:                              then type:
+---------------------------------------- ------------------------------------------
+a bash-compatible shell (bash/zsh)       ``source util/quickstart/setchplenv.bash``
+a csh-compatible shell (csh/tcsh)        ``source util/quickstart/setchplenv.csh``
+the fish shell (fish)                    ``. util/quickstart/setchplenv.fish``
+the Bourne shell (sh)                    ``. util/quickstart/setchplenv.sh``
+======================================== ==========================================
+
+
+You must be in the top-level Chapel directory for these command
+scripts to work properly.  Note that there is no requirement to use
+these scripts long-term, they are merely designed to get new users
+up and running quickly.  See ``doc/chplenv.rst`` for a complete
+description of Chapel's environment variables and their options.
+
+If you use a shell other than the ones listed above and are able to
+create an equivalent ``util/quickstart/setchplenv.*`` script for it,
+please consider contributing a copy of it back to the project for
+inclusion in the next release.
+
```

Comment:
The two paragraphs above don't seem nicely indented in the GitHub view of the file.  I was guessing that this might just be to avoid the ugly '| grey' style indenting, but noticed that for the third item, you seem to have gotten both the indentation and the nice formatting somehow...